### PR TITLE
Clearer documentation

### PR DESCRIPTION
**Pull request by gagou9**
plus some weird merging stuff I don't get.

### CODE REVIEW

**Comment by lgarron** [comment]
Could you debase so that we... don't have weird merging stuff? :-P

File: README.md [hunk 1]
```diff
@@ -3,26 +3,27 @@ worldcubeassociation.org
...
+- SSH into the machine, with agent-forwarding enabled : `ssh -A you@yourmachine`
+- Provison your machine with `sudo wget https://raw.githubusercontent.com/cubing/worldcubeassociation.org/master/scripts/wca-bootstrap.sh -O /tmp/wca-bootstrap.sh && sudo -E bash /tmp/wca-bootstrap.sh [environment]` `[environment]` must be one of `development`, `development-noregs`, `staging`, `production`. This step will take a very long time (at least an hour)
+  - For the `staging` and `production` environment, you will need an SSH key set up or know the password for cubing@worldcubeassociation.org in order to rsync secrets.
+  - The bootstrapping part may fail while trying to install `texlive-lang-all`, in that case you can run `sudo apt-get install texlive-lang-all=2013.20140215-1` manually and then continue the bootstrapping process with `sudo -E bash /tmp/wca-bootstrap.sh [environment]`.
+- The website should now being served on port 80. The traffic will be redirected to https (443) for `staging` and `production` environments.
 
-## Deploy
```
Comment (jfly) [inline README.md]: I don't think "local" is the right word here.

File: README.md [hunk 2]
```diff
@@ -3,26 +3,27 @@ worldcubeassociation.org
...
+- Make sure SSH Agent is running or start it with `eval "$(ssh-agent -s)"`
+- Load your GitHub key into the ssh agent by doing `ssh-add /path/to/key`. If you don't have an SSH key, check the [GitHub howto](https://help.github.com/articles/generating-ssh-keys/)
+- SSH into the machine, with agent-forwarding enabled : `ssh -A you@yourmachine`
+- Provison your machine with `sudo wget https://raw.githubusercontent.com/cubing/worldcubeassociation.org/master/scripts/wca-bootstrap.sh -O /tmp/wca-bootstrap.sh && sudo -E bash /tmp/wca-bootstrap.sh [environment]` `[environment]` must be one of `development`, `development-noregs`, `staging`, `production`. This step will take a very long time (at least an hour)
+  - For the `staging` and `production` environment, you will need an SSH key set up or know the password for cubing@worldcubeassociation.org in order to rsync secrets.
+  - The bootstrapping part may fail while trying to install `texlive-lang-all`, in that case you can run `sudo apt-get install texlive-lang-all=2013.20140215-1` manually and then continue the bootstrapping process with `sudo -E bash /tmp/wca-bootstrap.sh [environment]`.
```
Comment (jfly) [inline README.md]: This is worrisome. Is something timing out? It would be nice to solve this issue if possible.

**Comment by jfly** [comment]
Looks good to me! One nitpick: could you add periods to the end of each sentence? 

All the instructions about ssh agent forwarding look pretty scary/complicated, unfortunately. Maybe we could update our bootstrap script to check these things and output useful error messages instead?

**Comment by jfly** [comment]
Closing this PR. @gagou9, feel free to reopen if you want to get this pushed over the finish line.

**Pull request closed without merging**